src/main/java/seedu/momentum/ui/StatCard.java

```diff
@@ -25,20 +23,27 @@
     /**
      * Creates a {@code StatCard} with the given {@code statisticEntry} to display.
      */
-    //public StatCard(StatisticEntry statisticEntry) {
-    //    super(FXML);
-    //    this.statisticEntry = statisticEntry;
-    //    label.setText(statisticEntry.getLabel());
-    //    value.setText(Double.toString(statisticEntry.getValue()));
-    //}
-
-
-    // TEMPORARY METHOD FOR TESTING
-    public StatCard(String[] statisticEntry) {
+    public StatCard(StatisticEntry statisticEntry) {
         super(FXML);
         this.statisticEntry = statisticEntry;
-        label.setText(statisticEntry[0]);
-        value.setText(statisticEntry[1]);
+        label.setText(statisticEntry.getLabel());
+        value.setText(formatToString(statisticEntry.getValue()));
+    }
+
+    private String formatToString(double value) {
+        String output = "";
+        int hours = (int) Math.floor(value / 60);
+        int minutes = (int) value % 60;
+
+        if (hours > 0) {
+            output += String.format("%d hr ", hours);
+        }
+
+        if (minutes > 0) {
+            output += String.format("%d min", minutes);
+        }
+
+        return output;
     }
```

Comment:
Do you think this method will be used elsewhere in the future? If it is, it might be a good idea to move it over to `StringUtil.java`

Reply:
I agree with Praveen, i think we can move it over to StringUtil